### PR TITLE
update for chalk

### PR DESCRIPTION
**Pull request by ghost**
Update for pebble round

### CODE REVIEW

File: src/feature_draw_bitmap_composite.c [hunk 1]
```diff
@@ -29,14 +29,17 @@ static void layer_update_callback(Layer *layer, GContext *ctx) {
...
   // Draw the large circle the image will composite with
   graphics_context_set_fill_color(ctx, GColorBlack);
   graphics_fill_circle(ctx, GPoint(bounds.size.w / 2, bounds.size.h + 110), 180);
 
   // Use the image size to help center the image
-#ifdef PBL_PLATFORM_BASALT
+#ifdef PBL_SDK_3
   GRect destination = gbitmap_get_bounds(s_image);
 #else
   GRect destination = s_image->bounds;
```
Comment (C-D-Lewis) [inline src/feature_draw_bitmap_composite.c]: This case and the `#ifdef` can be removed now.

**Comment by C-D-Lewis** [comment]
The image is a little high on Basalt due to the absence of the status bar. You can either correct for this in code, or remove the status bar on Aplite with `window_set_fullscreen()`, since it is of no relevance to the example.

**Comment by ghost** [comment]
@C-D-Lewis Code review changes have been made: fullscreen for aplite with better y offset for both aplite and basalt.

**Comment by C-D-Lewis** [comment]
:+1:

**Pull request merged**